extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -182,6 +185,13 @@ export class AmpA4A extends AMP.BaseElement {
   }
 
   /** @override */
+  buildCallback() {
+    const adType = this.element.getAttribute('type');
+    this.config = adConfig[adType];
+    user().assert(this.config, `Type "${adType}" is not supported in amp-ad`);
```

Comment:
This breaks ad type 'fake'.  In general, shouldn't this be checking _a4a_config instead?